tests/unittests/CoreGraphics/CGFontTests.mm

```diff
@@ -100,4 +100,51 @@
         EXPECT_EQ(expectedBoxes[i].size.width, boxes[i].size.width);
         EXPECT_EQ(expectedBoxes[i].size.height, boxes[i].size.height);
     }
+}
+
+TEST(CGFont, CreateWithDataProvider) {
+    // get test startup full path
+    wchar_t fullPath[_MAX_PATH];
+    size_t len = GetModuleFileNameW(NULL, fullPath, _MAX_PATH);
+
+// Window needs extra handling for the drive character, OSX has twice-as-wide wchars
```

Comment:
>Window [](start = 3, length = 6)

Not sure this comment is clear.  In theory we should be able to use stringByDeletingLastPathComponent on windows just as well.